openimis.json

```
@@ -152,9 +152,13 @@
 			"name": "controls",
 			"pip": "git+https://github.com/openimis/openimis-be-controls_py.git@develop#egg=openimis-be-controls"
 		},
-	        {
-	            "name": "grievance",
-	            "pip": "git+https://github.com/openimis/openimis-be-grievance_py.git@develop#egg=openimis-be-grievance"
-	        }
+		{
+			"name": "grievance",
+			"pip": "git+https://github.com/openimis/openimis-be-grievance_py.git@develop#egg=openimis-be-grievance"
+		},
+		{
+			"name":"dashboard_etl",
+			"pip" : "git+https://github.com/openimis/openimis-be-dashboard_etl_py.git@develop#egg=openimis-be-dashboard_etl"
+		}
```

```suggestion
		}
```
we are not ready to add it in this release, second database crashes most of the tests